stdlib/contrib/chobbs/discord/discord.flux

```diff
@@ -0,0 +1,21 @@
+package discord
+
+import "http"
+import "json"
+
+// `url` - string - URL of the discord webhook endpoint
+// `username` - string - Username posting the message.
+// `content` - string - The text to display in discord.
+
+send = (url, botuser, content) => {
```

Comment:
yeah, I think that is pretty important to avoid injection style attacks.  @Opus-the-penguin -- comments?

Reply:
@rockstar  @Opus-the-penguin  -thanks agin for the feedback Paul, glad to make the suggested update here - but would call out ive look at other api implications and they all seem to follow the pattern of just accepting a single URL string including the root. This may be due to the fact that discord makes it easily consumable a as one string